### PR TITLE
Feature Layer Category Modularization

### PR DESCRIPTION
**Pull request by jenmerritt**
### Description

- modularizes the feature layer category

Checklist:
- [ ] Set target branch to main (current release) or v.next (next release)
- [ ] Request a reviewer
- [ ] Assign a reviewer
- [ ] Tag reviewer in comment

### CODE REVIEW

**Comment by jenmerritt** [comment]
Ready for review @Luiy0

File: feature_layers/feature-layer-query/src/main/java/module-info.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+module com.esri.samples.feature_layer_query {
+  // require ArcGIS Runtime module
+  requires com.esri.arcgisruntime;
+
+  // require JavaFX modules that the application uses
+  requires javafx.graphics;
+  requires javafx.controls;
+
+  exports com.esri.samples.feature_layer_query;
+}
```
Comment (Luiy0) [inline feature_layers/feature-layer-query/src/main/java/module-info.java]: Newline here.

File: feature_layers/list-related-features/src/main/java/module-info.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+module com.esri.samples.list_related_features {
+  // require ArcGIS Runtime module
+  requires com.esri.arcgisruntime;
+
+  // require JavaFX modules that the application uses
+  requires javafx.graphics;
+  requires javafx.controls;
+
+  exports com.esri.samples.list_related_features;
+}
```
Comment (Luiy0) [inline feature_layers/list-related-features/src/main/java/module-info.java]: Newline here.

**Review by Luiy0 — APPROVED** [review]
LGTM @jenmerritt

**Pull request merged**